README.md

```diff
@@ -1,284 +1,522 @@
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird Desk SDK for JavaScript.
+
+### Requirements
+
+- `NodeJS 10.13.x+ along with npm`
+- [`Sendbird JavaScript SDK 3.0.55 or later`](https://github.com/sendbird/Sendbird-JavaScript)
+
+### Supported browsers
+
+- `All modern versions of Chrome, FireFox, Edge, Safari, and etc. supporting ES6+`
+- `IE 11+`
+- `Mobile browsers (Android/iOS)`
+
+<br />
+
+## Getting started
+
+This section gives you information you need to get started with Sendbird Desk SDK for JavaScript. 
+
+### Try the sample app
+
+Our sample app demonstrates the core features of Sendbird Desk SDK. Download the app from our GitHub repository to get an idea of what you can do with the actual SDK and to get started building your own project.
+ 
+- https://github.com/sendbird/quickstart-desk-javascript
+
+### Step 1: Create a Sendbird application from your dashboard
+
+A Sendbird application comprises everything required in a chat service including users, message, and channels. To create an application:
+
+1. Go to the Sendbird Dashboard and enter your email and password, and create a new account. You can also sign up with a Google account.
+2. When prompted by the setup wizard, enter your organization information to manage Sendbird applications.
+3. Lastly, when your dashboard home appears after completing setup, click **Create +** at the top-right corner.
+
+Regardless of the platform, only one Sendbird application can be integrated per app; however, the application supports communication across all Sendbird’s provided platforms without any additional setup. 
+
+> **Note**: All the data is limited to the scope of a single application, thus users in different Sendbird applications are unable to chat with each other.
+
+### Step 2: Download and install the Desk SDK
+
+Installing the Desk SDK is simple if you’re familiar with using external libraries or SDK’s in your projects. These are the following three methods for Desk SDK implementation: 
+
+Install the Desk SDK with npm by entering the command below on the command line.
+
+```bash
+# npm
 ~$ cd /path/to/your-project
 ~$ npm install --save sendbird-desk
 ```
 
-> You can use SendBird SDK along with SendBird Desk SDK (e.g. you can build your own messaging service while you use Desk SDK as well) In that case, you must use **the same APP_ID** for both SDKs. And please note that SendBird SDK version must be 3.0.55 or higher to work with Desk SDK.
+Automatically download our Desk SDK with just an npm install command by adding sendbird-desk dependency to the package.json file of your project.
+ 
+Download the latest Desk SDK for JavaScript, copy the Desk SDK to your project classpath (most commonly the lib/ directory), and then include the Desk SDK file to your working file.
+
+```javascript
+<script src="lib/SendBird.min.js"></script>
```

Comment:
This should be the Desk SDK, not the core SDK.

Reply:
Done!